### ISSUE TITLE
Issues with alphas_cumprod and scheduling when trying epsilon prediction

### ISSUE THREAD

**Issue by hlillemark** [kwsong0113/diffusion-forcing-transformer#11]
Hi, I had some questions about using different objectives for training: It seems all the default configurations have used v prediction, which can avoid some instability with alphas_cumprod becoming too small when k is close to K (and thus 1/alphas_cumprod exploding). When I try to train a model with epsilon prediction (just the same as the v_prediction model on minecraft data, with the image vae latent space), the sampling always looks terrible, despite the loss converging. Even if the model is not fully converged, I would expect the samples to look fuzzy, not be just a black screen due to out of bound predictions. The code for epsilon prediction looks more or less standard, it's just not implemented for continuous diffusion so I used discrete diffusion. After some manual inspection of the magnitudes of the predictions at high noise levels in DDIM, it shows that the std of the predicted x can be >>1, which after all the sampling steps, causes the magnitudes to explode. 

I was wondering if the authors have tested this model and training scheme with other types of predictions, and whether there is some insight into why this training method would be significantly different than others? Manually inspecting some other trained models revealed large magnitude of 1/alphas_cumprod, but still a stable std of ~= 1 for all the sampling steps of DDIM. And if I may ask, are there any insights into how the different diffusion hparams (prediction type, noise schedule, loss weighting) was chosen in the end?

Additionally, are there some insights into how to adjust the learning rate when the effective batch size increases? E.g. training on 1 node of 8xa100 to two nodes of 8xa100? 

Thank you!

**Comment by buoyancy99** [comment]
We haven't ablated epsilon objective a lot, so unfortunately I cannot tell immediately whether this is a big. Usually for domains where high frequency detail is important we always use pred_v following other image diffusion papers.

Learning Rate: Our original learning rate was designed for 12 GPUs, so you simply multiply original lr by 0.666 or 0.5 since your batch size is 2/3 of the original

**Comment by hlillemark** [comment]
Ok, thanks for the info!
